peekaboo/toolbox/file.py

```diff
@@ -32,48 +32,49 @@
 logger = logging.getLogger(__name__)
 
 
-def guess_mime_type_from_file_contents(file_path):
-    """  Get type from file magic bytes. """
-    mt = magic.from_file(file_path, mime=True)
-    if not mt:
-        return None
+class Filetools(object):
+    """ Parent class, defines interface to various file tools. """
 
-    return mt
+    def guess_mime_type_from_file_contents(self, file_path):
```

Comment:
The intention here being to make it clear that FileTools in their role as analyser are now in charge of determining these things and there are and should be no external users who don't go through FileToolsReport anymore?

Reply:
Addressed in #143.